.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+## Changes:
+<!--- Provide the broad strokes of the changes made in a bulleted list --->
+
+-
+
+## Compatiblity:
+
+Any changes that require a newer version of the API than the oldest currently 
+supported version are considered breaking changes.
+
+Common breaking changes:
+ -  Using a new API not available in oldest currently supported server
+ -  Requiring new elements in an existing response from the API
+ -  Dropping support for a previously valid response model from the API.
```

Comment:
It took me a while to figure out where you were getting at with this. The other ones are obvious, but I think this needs a brief example.